apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -719,7 +723,7 @@ export const getEntityAuthorizationRelationships: ImpureGraphFunction<
       data.map(
         (relationship) =>
           ({
-            object: params.entityId,
+            object: { namespace: "entity", id: params.entityId },
```

Comment:
We discussed and will rename:
1. `namespace` to `kind` (for consistency with other discriminating fields, e.g. in the subgraph)
2. `id` to `objectId` / `subjectId` (avoid unqualified `id`s)

Reply:
- Renamed `namespace` to `kind` in [`d6089df` (#3423)](https://github.com/hashintel/hash/pull/3423/commits/d6089dfd288b8dfc291262968fc6c781864d2dcc)
- Renamed `id` to `objectId` and `subjectId`: [`17f8f18` (#3423)](https://github.com/hashintel/hash/pull/3423/commits/17f8f182f15bf63e0165ec914e5103c20c7dba4c) (I'm not 100% convinced, seems a bit verbose but it could be helpful in some contexts). cc @benwerner01